internal/assets/contents/activestate.yaml.perl.tpl

```diff
@@ -44,4 +44,8 @@ events:
   # This is the ACTIVATE event, it will run whenever a new virtual environment is created (eg. by running `state activate`)
   # On Linux and macOS this will be ran as part of your shell's rc file, so you can use it to set up aliases, functions, environment variables, etc.
   - name: ACTIVATE
+    if: ne .Shell "cmd"
+    value: {{.LangExe}} $scripts.activationMessage.path._posix()
+  - name: ACTIVATE
+    if: eq .Shell "cmd"
```

Comment:
What's the relevance to this PR?

Reply:
If you activate a project in the Git Bash shell on Windows, the ACTIVATE event will try and use a cmd-style path instead of a bash-style path, resulting in a "path not found" type of error.

Reply:
I see. Then I would ask for a change here too; bash support for windows mean that the end-user doesn't need to work around limitations with _posix(). So what you did here should be a non-issue if we supported bash.

Reply:
I wanted to do this, but the ScriptExpander doesn't have access to the underlying subshell. This would possibly be a significant change. I'll have to revisit this (it's been almost a week since I looked into it).

Reply:
It looks like I managed to work in the subshell into the Expander context.